style/pages/projects-details.css

```diff
@@ -15,13 +18,78 @@
 }
 
 .projects-details .related-projects-list .searchbar,
-.projects-details .related-projects-list .nav > * {
+.projects-details .related-projects-list .projects > * {
   border-bottom: 1px solid var(--color-border);
 }
 
-.projects-details .related-projects-list .nav {
+.projects-details .related-projects-list .projects {
   overflow-y: auto;
-  height: calc(100vh - 130px);
+  height: calc(100vh - (var(--app-header-height) + 64px));
+}
+
+.projects-details .container {
+  --container-padding: 1.8rem;
+  gap: 1rem;
+  overflow-x: hidden;
+}
+
+.projects-details .project .header,
+.projects-details .project .intro .stats,
+.projects-details .project,
+.projects-details .activity {
+  display: flex;
+}
```

Comment:
I see you combine same style here, but why you don't do this thing in other files?